### ISSUE TITLE
Varianssianalyysi

### ISSUE THREAD

**Issue by jukop** [jukop/r_intro#3]
Varianssianalyysiin voisi lisätä pienen teoriaosuuden, jossa selitettäisiin vapausasteet neliösummamäärittelyiden kautta.
Ks. esim. https://en.wikipedia.org/wiki/F-test

**Comment by santikka** [comment]
Totta, mutta tehtävän idea onkin testata ymmärrystä siitä, kumpi vapausaste liittyy taustamuuttujaan.
